go-runtime/modulecontext/module_context_test.go

```diff
@@ -21,13 +21,13 @@ func TestConfigPriority(t *testing.T) {
 
 	moduleName := "test"
 
-	cp := cf.NewInMemoryProvider[cf.Configuration]()
+	cp := cf.InlineProvider[cf.Configuration]{}
```

Comment:
Use `cf.InlineProvider[cf.Configuration]{Inline: true}` to fix the test failure IIRC.

Reply:
Just saw the same thing 👍